addon/data/shooter-interactive-worker.js

```diff
@@ -273,6 +281,9 @@ stateHandlers.crosshairs = {
   },
 
   mousedown: function (event) {
+    if (event.target.className === "pageshot-myshots") {
+      return;
+    }
```

Comment:
The functionality was only needed in Chrome because it had the in-content top bar, but the idea is the same.  Maybe move `isHeader` into a standalone function that handles both cases?